Automattic/wp-calypso
# Improve cancellation flow

#### Proposed Changes

* This PR improves the cancellation flow as part of the Cancellation Flow project. See pebzTe-kA-p2 for more details.

#### Testing Instructions



* Purchase a dotcom plan.
* Try to cancel it by clicking `Remove plan` on `My Home > Purchases`.
  <img width="1087" alt="Purchase_Settings_‹_Cancel_Test_—_WordPress_com" src="https://user-images.githubusercontent.com/212034/195392169-d937c51f-b07d-4fa6-a270-b3fc4fc406c6.png">
* Pick options and fill all forms.
  * You may see an additional form to ask your next adventure.
     <img width="800" alt="Purchase_Settings_‹_Cancel_Test_—_WordPress_com" src="https://user-images.githubusercontent.com/212034/195393028-ad68b31a-b34e-43d0-b762-8f978d052f4f.png">
  * Or the upsell form will show up instead. Try to cancel Premium plan and pick `Price/Bugdget` and `It's too expensive.`
    <img width="788" alt="Cancel_Purchase_‹_Cancel_Test_—_WordPress_com" src="https://user-images.githubusercontent.com/212034/195798788-2696338e-6c46-4dbb-b14d-a0546ad6a975.png">

    
* This flow should work with Atomic sites as well.

#### Pre-merge Checklist






Related to #
